buildSrc/src/test/resources/edu/ucar/build/ui/netCDFtoolsExtraJars.jnlp

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<jnlp spec="7.0" codebase="http://www.unidata.ucar.edu/software/thredds/current/netcdf-java/webstart"
+        href="netCDFtoolsExtraJars.jnlp" version="1.5">
+    <component-desc/>
+    
+    <resources>
+        <jar href='root-1.5.jar' main='true' download='eager' />
+    
+        <jar href='slf4j-api-1.7.7.jar' download='lazy' />
+        <jar href='objenesis-2.2.jar' download='lazy' />
+        <jar href='hamcrest-core-1.3.jar' download='lazy' />
```

Comment:
This file is only a test resource (a "control" file used for comparison -- look at its path). I know that's confusing, because there used to be a file with the same name (but slightly different path) that we edited and then published every release. In any event, the versions of these dependencies are what they are because of an integration test.